cauta-firma.html

```diff
@@ -75,42 +75,268 @@
                                 <div class="navbar-header navbar-header-custom">
                                     <!-- start logo -->
                                     <a href="index.html"
-                                       class="navbar-brand width-200px sm-width-180px xs-width-150px"><img
-                                            id="logo" src="img/logos/logo.png" alt="logo"></a>
+                                       class="navbar-brand width-200px sm-width-180px xs-width-150px">
+                                        <img id="logo" src="img/logos/Asset%2012@2x.png" alt="logo"></a>
```

Comment:
Te rog sa redenumesti logo-ul vechi (logo.png), in `old_logo.png`
Te rog sa redenumesti logo-ul din `Asset%2012@2x.png` in `logo.png`